packages/date/package.json

```diff
@@ -0,0 +1,26 @@
+{
+	"name": "@wordpress/date",
+	"version": "0.0.1",
+	"description": "Date module for WordPress",
+	"author": "WordPress",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"wordpress",
+		"date"
+	],
+	"homepage": "https://github.com/WordPress/gutenberg/tree/master/packages/date/README.md",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/WordPress/gutenberg.git"
+	},
+	"bugs": {
+		"url": "https://github.com/WordPress/gutenberg/issues"
+	},
+	"dependencies": {
+		"moment": "2.22.1",
+		"moment-timezone": "0.5.16"
+	},
+	"publishConfig": {
+		"access": "public"
+	}
+}
```

Comment:
I believe this is lacking the `main/module` configs to make the modules ready for npm. This would probably have an impact on the webpack config (separate dist folder) than the one currently used for the remaining modules

Reply:
Which brings us back to having a dedicated build folder per package maybe 🤷‍♂️

Reply:
It seems like we need to have independent build folders for npm, because we use very specific Webpack config which does some additional work like exposing `wp.` globals. We also need to expose two version:
- es5 friendly
- import/export friendly

Reply:
I've always found a dedicated build folder much easier to grok when I first explore a package